rosdep/python.yaml

```
@@ -5813,6 +5813,18 @@ python3-events-pip:
   ubuntu:
     pip:
       packages: [Events]
+python3-ezdxf:
+  debian: [python3-ezdxf]
```

This seems to only be available for Bullseye and newer.

```suggestion
  debian:
    '*': [python3-ezdxf]
    buster:
      pip:
        packages: [ezdxf]
    stretch:
      pip:
        packages: [ezdxf]
```